recipes/oracle-jdk/post-link.bat

```diff
@@ -0,0 +1,46 @@
+
+@echo off
+
+echo "Installing in %CONDA_PREFIX%" > "%CONDA_PREFIX%\.messages.txt"
+echo "   CONDA_PREFIX: %CONDA_PREFIX%" >> "%CONDA_PREFIX%\.messages.txt"
+echo "   PKG_NAME:     %PKG_NAME%"     >> "%CONDA_PREFIX%\.messages.txt"
+echo "   PKG_VERSION:  %PKG_VERSION%"  >> "%CONDA_PREFIX%\.messages.txt"
+echo "   PKG_BUILDNUM: %PKG_BUILDNUM%" >> "%CONDA_PREFIX%\.messages.txt"
+:: set | FindStr "PKG" >> "%CONDA_PREFIX%\.messages.txt"
+
+set "PKG_BIN=%CONDA_PREFIX%\bin"
+set "PKG_UUID=%PKG_NAME%-%PKG_VERSION%_%PKG_BUILDNUM%"
+
+:: should I be using `reg query ` to find the jdk directory path?
+set "ORACLE_JDK_DIR=%ProgramFiles%\Java\jdk1.8.0_321"
```

Comment:
I will be needing to change discovery to work with any (latest) release of a particular version as it changes fairly regularly.
Rather than be specific about jdk1.8.0_321 the discovery should pick the latest revision; only match on the prefix jdk1.8.0